ledger/src/blockstore_meta.rs

```diff
@@ -13,7 +13,7 @@ use {
 };
 
 bitflags! {
-    #[derive(Deserialize, Serialize)]
+    #[derive(Copy, Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
I was curious about the change to derived trait implementations.

I see that 1.3.2 automatically derived several traits:
https://docs.rs/bitflags/1.3.2/src/bitflags/lib.rs.html#136-139

Whereas 2.3.3 does not:
https://docs.rs/bitflags/2.3.3/src/bitflags/lib.rs.html#215-216
https://docs.rs/bitflags/2.3.3/src/bitflags/lib.rs.html#316-317

Hence, the change to manually specify